src/reaper/wait.rs

```diff
@@ -217,6 +217,53 @@ cfg_if::cfg_if! {
             }
         }
 
+        /// Tell if we are able to use this backend.
+        pub(crate) fn available() -> bool {
+            true
+        }
+    } else if #[cfg(target_os = "freebsd")] {
+        use async_io::os::kqueue::{Exit, Filter};
+        use std::num::NonZeroI32;
+
+        /// Waitable version of `std::process::Child`
+        struct WaitableChild {
+            child: std::process::Child,
+            handle: Filter<Exit>,
+        }
+
+        impl WaitableChild {
+            fn new(child: std::process::Child) -> io::Result<Self> {
+                // std::process::Child id must provide a positive PID value
+                let exit_filter = unsafe  { Filter::new(Exit::from_pid(NonZeroI32::new_unchecked(
+                        child
+                        .id()
+                        .try_into()
+                        .expect("could not transform pid to i32 type")
+                    )))?
+                };
```

Comment:
Can you reformat this to match rusfmt formatting? Unfortunately rustfmt doesn't auto-format code in macro blocks.